examples/quantizing_moe/deepseek_moe_w4a16.py

```diff
@@ -5,6 +5,10 @@
 from llmcompressor.transformers import oneshot
 from llmcompressor.transformers.compression.helpers import calculate_offload_device_map
 
+# NOTE: transformers 4.48.0 has an import error with DeepSeek.
+# Please consider either downgrading your transformers version to a
+# previous version or upgrading to a version where this bug is fixed
```

Comment:
Should include recommended version 
4.47.1 works

Reply:
I think this is overkill, especially as we’re suggesting downgrading